dask/dataframe/tests/test_rolling.py

```diff
@@ -166,6 +166,29 @@ def test_rolling_methods(method, args, window, center, check_less_precise):
     )
 
 
+if PANDAS_VERSION <= "0.25.0" and PANDAS_VERSION >= "0.20.0":
```

Comment:
Does this need to be conditional on the pandas version? I don't see any harm in always applying the warning.

Reply:
I was thinking that after >= 0.25.0 will be used for all CI test envs this whole thing (```filter_panel_warning```) could be removed, right now dask-35.yml is failing without this. By "always applying the warning" did you mean - let CI fail?

Reply:
Sorry I meant "always apply the warnings filter". Not a big deal though.